bin/postman-test/collections/system-management.postman_collection.json

```diff
@@ -273,19 +281,23 @@
           "raw": ""
         },
         "url": {
-          "raw": "{{systemManagementUrl}}/api/v1/ping",
+          "raw": "{{systemManagementUrl}}/api/v1/metrics/device-virtual,edgex-core-command,edgex-core-data,edgex-core-metadata,edgex-support-logging,edgex-support-notifications,edgex-support-scheduler",
+          "protocol": "http",
           "host": [
-            "{{systemManagementUrl}}"
+            "localhost"
```

Comment:
The host should set to variable {{systemManagementUrl}}.

Reply:
I have addressed this and pushed the update 👍